recipes/libbacktrace/all/patches/0002-msvc-unistd-alternative.patch

```diff
@@ -0,0 +1,54 @@
+--- a/allocfail.c
```

Comment:
Do you intend to upstream this patch?

Reply:
Not sure. I don't know if it really makes sense to support Visual Studio. I didn't fix all the files actually, only the ones built when libunwind is not found.